vcd/resource_vcd_vapp.go

```diff
@@ -282,7 +323,7 @@ func resourceVcdVAppCreate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
-	d.SetId(d.Get("name").(string))
+	d.SetId(vapp.VApp.ID)
```

Comment:
Looks like we're switching from saving name to ID. Won't these cause unexpected regressions? If not, we still need to tell this in the change log as we did with others.

Reply:
We have done this for all resources in which we have implemented import and data sources.
I will add to the change log

Reply:
done